metrics/report/makereport.sh

```diff
@@ -23,6 +23,12 @@ GUESTINPUTDIR="/inputdir/"
 GUESTOUTPUTDIR="/outputdir/"
 
 setup() {
+	echo "Checking subdirectories"
+	check_subdir="$(cd ${HOSTINPUTDIR}; ls -dx */ | wc -l)"
```

Comment:
I think we can send the `stderr` to `/dev/null` here.
I see that this line may send this error:
```
ls: cannot access '*/': No such file or directory
```